setup.py

```diff
@@ -32,8 +32,8 @@
       cmdclass=versioneer.get_cmdclass(),
       description='String parser/formatter',
       long_description=README,
-      author='Panu Lahtinen',
-      author_email='panu.lahtinen@fmi.fi',
+      author='The Pytroll Team',
+      author_email='pytroll@googlegroups.com',
```

Comment:
@adybbroe the problem with this is that only people subscribed to the mailing list can send an email there, right ?

Reply:
@adybbroe what do you think we should do here?

Reply:
I think we started having this conversation a year ago when this PR was first made. We never really came to a solution. We could make a `groups.io` maintainer email list? Or designate someone as **the** core maintainer of a package and put their email.

Reply:
How about using slack? https://slack.com/intl/en-se/help/articles/206819278-Send-emails-to-Slack

Reply:
That's an interesting idea. A little scary, but interesting. If you want to try it on one of the pytroll packages and see how long it takes to get spam then :+1:

Reply:
I would leave it with the pytroll google group, that is how we have in Satpy, and it minimizes the risk of spam. Anyhow, it is really not difficult to get in touch with Pytroll, either via slack or mail.
Ok @djhoese and @mraspaud ?